### ISSUE TITLE
8.7.2 not working on Alpine

### ISSUE THREAD

**Issue by mlarcher** [TanStack/table#4604]
### Describe the bug

When upgrading @tanstack/react-table to 8.7.2 we have an error message in the console and no table is being displayed when bulding the app in an Alpine Docker container (node:18.10.0-alpine). It works fine in 8.7.0 (8.7.1 is not valid on npm it seems), and the bug does not appear when running in local dev on macos.

Here is the error we see :
![Capture d’écran 2022-12-13 à 11 28 24](https://user-images.githubusercontent.com/1116431/207293908-edf2e542-56f0-4063-bc37-2f70e9ad6967.png)
And here the code the error points to:
![Capture d’écran 2022-12-13 à 11 35 12](https://user-images.githubusercontent.com/1116431/207295052-bfc134fe-700d-40ee-9289-1769634a26cb.png)

Our app is based on Nextjs 13.

Please let me know if you need more info to fix the issue.

### Your minimal, reproducible example

not sure those tools can handle such a case

### Steps to reproduce

1. create a Nextjs React application using react-table
2. build it in an Alpine container
3. go to the page with react-table being served

### Expected behavior

the table displays correctly, no error appears in the console

### How often does this bug happen?

Every time

### Platform

node:18.10.0-alpine

### react-table version

8.7.2

### Terms & Code of Conduct

- [X] I agree to follow this project's Code of Conduct
- [X] I understand that if my bug cannot be reliable reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by polpenaloza** [comment]
Revert your stack to `v8.7.0` while they fix it, I lost 4 hours today with this.
```json
    "@tanstack/match-sorter-utils": "8.7.0",
    "@tanstack/react-table": "8.7.0",
    "@tanstack/table-core": "8.7.0",
```

**Comment by mlarcher** [comment]
reverting `@tanstack/react-table` to `8.7.0`  was enough for me. I don't explicitely require table-core and match-sorter-utils in 8.7.2 is fine.
I hope this get addressed quickly

**Comment by lukaszwisniewski88** [comment]
> Revert your stack to `v8.7.0` while they fix it, I lost 4 hours today with this.
> 
> ```json
>     "@tanstack/match-sorter-utils": "8.7.0",
>     "@tanstack/react-table": "8.7.0",
>     "@tanstack/table-core": "8.7.0",
> ```

Same for me.

**Comment by grahamorrell** [comment]
Same here, same error using NextJS 13 (but not Alpine). Reverting to 8.7.0 fixed it.

**Comment by lobatolais** [comment]
Does [8.7.3](https://github.com/TanStack/table/releases/tag/v8.7.3) fix this?

**Comment by mlarcher** [comment]
nope it still fails with 8.7.3

**Comment by tresorama** [comment]
Same here, 8.7.3 and Next.js 12.3.4.
Error appears only in prod, (`npm run build && npm run start`), in dev mode all works fine.
Reverted to 8.7.0 , fixed !

**Comment by dfmcphee** [comment]
We have the same issue with `8.7.3` and Next `13.0.0`.

**Comment by t-moennich** [comment]
We also have the same issue with `8.7.2`, `8.7.3` and Next.js (`13.0.5`).
Going back to `8.7.0` fixed it.

In our case, the error only happens with production build.
During development, it doesn't appear even with 100% same data.

**Comment by KevinVandy** [comment]
Try v8.7.4

**Comment by farreldarian** [comment]
> Try v8.7.4

Still doesn't work unfortunately

**Comment by KevinVandy** [comment]
> Same here, 8.7.3 and Next.js 12.3.4. Error appears only in prod, (`npm run build && npm run start`), in dev mode all works fine. Reverted to 8.7.0 , fixed !

This is strange because in my testing, there is no error with Table v8.7.4 and Next 12.3.4, but there is with Next 13

**Comment by mlarcher** [comment]
Still not working with 8.7.4 (using Next 13.0.6)

**Comment by The-Code-Monkey** [comment]
I have an issue similar to this but instead i get this error 
```Application error: a client-side exception has occurred (see the browser console for more information).```
This error only shows in prod but doesnt print anything to the browser console or server console.

**Comment by tresorama** [comment]
> > Same here, 8.7.3 and Next.js 12.3.4. Error appears only in prod, (`npm run build && npm run start`), in dev mode all works fine. Reverted to 8.7.0 , fixed !
> 
> This is strange because in my testing, there is no error with Table v8.7.4 and Next 12.3.4, but there is with Next 13

Tested now, Next.js 12.3.4 and React-table 8.7.4:
Same error in production.

**Comment by KevinVandy** [comment]
> > > Same here, 8.7.3 and Next.js 12.3.4. Error appears only in prod, (`npm run build && npm run start`), in dev mode all works fine. Reverted to 8.7.0 , fixed !
> > 
> > This is strange because in my testing, there is no error with Table v8.7.4 and Next 12.3.4, but there is with Next 13
> 
> Tested now, Next.js 12.3.4 and React-table 8.7.4:
> Same error in production.

As stated in the other issues, if you are using next 13, you can turn swcMinify to false, or downgrade react table to v8.7.0

**Comment by The-Code-Monkey** [comment]
> > > > Same here, 8.7.3 and Next.js 12.3.4. Error appears only in prod, (`npm run build && npm run start`), in dev mode all works fine. Reverted to 8.7.0 , fixed !
> > > 
> > > 
> > > This is strange because in my testing, there is no error with Table v8.7.4 and Next 12.3.4, but there is with Next 13
> > 
> > 
> > Tested now, Next.js 12.3.4 and React-table 8.7.4:
> > Same error in production.
> 
> As stated in the other issues, if you are using next 13, you can turn swcMinify to false, or downgrade react table to v8.7.0

Ok I understand this is the workaround but who is actually causing the issues the swcminifier or this repo?

**Comment by OmerWow** [comment]
> > > > > Same here, 8.7.3 and Next.js 12.3.4. Error appears only in prod, (`npm run build && npm run start`), in dev mode all works fine. Reverted to 8.7.0 , fixed !
> > > > 
> > > > 
> > > > This is strange because in my testing, there is no error with Table v8.7.4 and Next 12.3.4, but there is with Next 13
> > > 
> > > 
> > > Tested now, Next.js 12.3.4 and React-table 8.7.4:
> > > Same error in production.
> > 
> > 
> > As stated in the other issues, if you are using next 13, you can turn swcMinify to false, or downgrade react table to v8.7.0
> 
> Ok I understand this is the workaround but who is actually causing the issues the swcminifier or this repo?

Hi,

SWCMinify is indeed what's causing this issue.

You can read my comment over here https://github.com/TanStack/table/issues/4610#issuecomment-1357263699
Also, I've opened an issue for Vercel (referenced in the issue I've linked above) the but haven't received a reply yet.

**Comment by mlarcher** [comment]
as stated in https://github.com/TanStack/table/issues/4610#issuecomment-1363218833 NextJS 13.1.0 fixes the issue.

**Comment by KevinVandy** [comment]
Table v8.7.6 should also fix this for the non-next 13.1 users.
